### ISSUE TITLE
field editing popups

### ISSUE THREAD

**Issue by tobibeer** [Jermolene/TiddlyWiki5#1448]
1. when adding a new field, show a popup of existing (custom) field names
2. when selected set the title of the field to be added to the selected one
3. for the field value, query against possible values for the specified field-name and allow the user to select from the list

**Comment by tobibeer** [comment]
I was doing a test for 1+2...

http://fields.tiddlyspot.com/#FieldsPopup

However I run into 2 issues:

1. the popup pops up but immediately closes when the input has focus
2. the field is directly added rather than merely set to the input field

**Comment by pmario** [comment]
should be part of 5.1.14?

**Comment by tobibeer** [comment]
@pmario: no, an ability to equally select a field value equivalent to the field name does not exist. Renaming the issue.

**Comment by tobibeer** [comment]
This may indeed depend on / require some considerations along the lines of #1331, e.g. `type=select`.

**Comment by pmario** [comment]
1) and 2) are possible now and 3) can be implemented using: https://tiddlywiki.com/#Customizing%20EditTemplate%20field%20rendering
